#343: EffectiveCost clarification (#350)
FinOps-Open-Cost-and-Usage-Spec/FOCUS_Spec: specification/columns/effectivecost.md

# Effective Cost

> Effective Cost represents a cost inclusive of the impacts of all reduced rates and discounts, augmented with the [*amortization*](#glossary:amortization) of relevant purchases (one-time or recurring) paid to cover future eligible charges. The *amortized* portion included should be proportional to the [Pricing Quantity](#pricingquantity) and the time granularity of the data. Effective Cost does not use rates blended from when a given service is charged by multiple rates due to variable pricing tiers (e.g., usage based pricing tiers). This cost is denominated in the [Billing Currency](#billingcurrency). The Effective Cost is commonly utilized to track and analyze spending trends.

Reviewer: @AWS-ZachErdman sentence seems to have a minor issue.. "from when a given..."

> Effective Cost represents a cost inclusive of the impacts of all reduced rates and discounts, augmented with the [*amortization*](#glossary:amortization) of relevant purchases (one-time or recurring) paid to cover future eligible charges. The *amortized* portion included should be proportional to the [Pricing Quantity](#pricingquantity) and the time granularity of the data. Effective Cost does not use rates blended from when a given service is charged by multiple rates due to variable pricing tiers (e.g., usage based pricing tiers). This cost is denominated in the [Billing Currency](#billingcurrency). The Effective Cost is commonly utilized to track and analyze spending trends.

Reviewer 2: This implies "blending" is a known concept. In my experience, most people don't know what "blending" means. Many think of it as something different than what is implied here which makes this feel very AWS-specific. I'm proposing a tweak to make it more globally inclusive, use FOCUS terms (e.g., "unit price"), and hopefully a little simpler/clearer.
```suggestion
Effective Cost represents a cost inclusive of the impacts of all reduced rates and discounts, augmented with the [*amortization*](#glossary:amortization) of relevant purchases (one-time or recurring) paid to cover future eligible charges. The *amortized* portion included should be proportional to the [Pricing Quantity](#pricingquantity) and the time granularity of the data. Effective Cost is based on the effective unit price for the SKU after pre-purchase commitment discounts have been applied and does not mix or "blend" cost across multiple charges. This cost is denominated in the [Billing Currency](#billingcurrency). The Effective Cost is commonly utilized to track and analyze spending trends.
```

Fwiw, I suspect this would be much clearer if we had EffectiveUnitPrice. Then we just require the math work and we're good. We need that here regardless.

Author: How about simplifying further and saying:

> Effective Cost does not mix or "blend" costs across multiple charges of the same service.

Reviewer 3: @AWS-ZachErdman please review if Michael's change is ok to commit.

Reviewer 2: Love it! Here's a direct suggestion so it's easy to commit 🙂
```suggestion
Effective Cost represents a cost inclusive of the impacts of all reduced rates and discounts, augmented with the [*amortization*](#glossary:amortization) of relevant purchases (one-time or recurring) paid to cover future eligible charges. The *amortized* portion included should be proportional to the [Pricing Quantity](#pricingquantity) and the time granularity of the data. Effective Cost does not mix or "blend" costs across multiple charges of the same service.. This cost is denominated in the [Billing Currency](#billingcurrency). The Effective Cost is commonly utilized to track and analyze spending trends.
```

Author: I proposed an alternative to his change above, I'll add it to a commit

Author: Committed - thank you!
